src/Compression/Deflater.cs

```diff
@@ -0,0 +1,186 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Buffers;
+using System.Diagnostics;
+using System.Runtime.InteropServices;
+using System.Security;
+
+using ObjCRuntime;
+
+namespace Compression
+{
+	/// <summary>
+	/// Provides a wrapper around the ZLib compression API.
```

Comment:
comment seems out of context

Reply:
Yeah, copy paster from corefx. Removing.